haas_websocket/main.py

```diff
@@ -57,33 +57,33 @@ def startWebsocket(publisher):
     ws_endpoint = os.getenv('HAAS_WSS_ENDPOINT')
     ws = websocket.create_connection(ws_endpoint+b_token)
     while running() == True:
-        checkPass = rest.checkPassword()
-        if checkPass == True:
-            checkToken = rest.checkToken()
-            if checkToken == True: # checks if local bearer_token is the same as the gcp token
-                try:
-                    result = ws.recv() 
-                    print(result)
-                    msg_type, published = filterMessage(result,publisher)
-                    if published == True:
-                        print(f"(INFO) Successfully pushed {msg_type} message pub/sub. \nFull Message: {result}")
-                    else:
-                        print(f"(INFO) Successfully recieved {msg_type} message, did not publish to pub/sub \nFull Message: {result}")
-                except KeyboardInterrupt:
-                    break
-            else:
-                ws.close()
-                b_token = rest.tokenUpdate() # updates local token from the secret manager
-                ws = websocket.create_connection(ws_endpoint+b_token) # restarts the websocket
-        else:
+        check_pass = rest.checkPassword()
+        check_token = rest.checkToken()
+        if check_pass == False:
             ws.close()
             rest.passwordUpdate() # updates local token from the secret manager
             b_token = rest.signIn()
             ws = websocket.create_connection(ws_endpoint+b_token) # restarts the websocket
+        if check_token == False:
```

Comment:
should these be else if statements? Or is it ok if it goes into the first two in a row?

Reply:
I think they should both be checked.